constants.toml

```diff
@@ -142,3 +142,50 @@ ELASTICITY_MULTIPLIER = 2
 # --- Hyperlane constants ---
 HYPERLANE_BRIDGE_DOMAIN = 5555
 # --- End Hyperlane constants ---
+
+
+BANK_DISCRIMINANT = 0
+ACCOUNTS_DISCRIMINANT = 1
+CHAIN_STATE_DISCRIMINANT = 2
+OPERATOR_INCENTIVES_DISCRIMINANT = 3
+BLOB_STORAGE_DISCRIMINANT = 4
+PROVER_INCENTIVES_DISCRIMINANT = 5
+SEQUENCER_REGISTRY_DISCRIMINANT = 6
+UNIQUENESS_DISCRIMINANT = 7
+HYPERLANE_DISCRIMINANT = 8
+ATTESTER_INCENTIVES_DISCRIMINANT = 9
+BLOB_SELECTOR_DISCRIMINANT = 10
+EVM_DISCRIMINANT = 11
+MAILBOX_DISCRIMINANT = 12
+INTERCHAIN_GAS_PAYMASTER_DISCRIMINANT = 13
+MERKLE_TREE_HOOK_DISCRIMINANT = 14
+WARP_DISCRIMINANT = 15
+PAYMASTER_DISCRIMINANT = 16
+REVENUE_SHARE_DISCRIMINANT = 17
+
+
+# Test modules
+MODULE_WITH_VERSIONED_STATE_ACCESS_IN_SLOT_HOOK_DISCRIMINANT = 233
+GAS_TESTER_DISCRIMINANT = 233
```

Comment:
MODULE_WITH_VERSIONED_STATE_ACCESS_IN_SLOT_HOOK_DISCRIMINANT and GAS_TESTER_DISCRIMINANT have the same discriminant